.circleci/config.yml

```diff
@@ -10,54 +12,36 @@ defaults: &defaults
 # restore_repo: &restore_repo
 #   restore_cache:
 #     key: v1-repo-{{ .Branch }}-{{ .Revision }}
-install_js: &install_js
-  run:
-    name: Install js dependencies
-    command: |
-      yarn config set yarn-offline-mirror ~/.cache/npm-packages-offline-cache/v1
-      yarn
-restore_yarn_cache: &restore_yarn_cache
-  restore_cache:
-    key: v2-yarn-sha-{{ checksum "yarn.lock" }}
-restore_yarn_offline_mirror: &restore_yarn_offline_mirror
-  restore_cache:
-    key: v2-npm-packages-offline-cache
-version: 2
+
+commands:
+  install_js:
+    steps:
+      - restore_cache:
+          keys:
+             - v2-yarn-sha-{{ checksum "yarn.lock" }}
+             - v2-yarn-sha-
+      - run:
+          name: Install js dependencies
+          command: yarn
+
 jobs:
   checkout:
     <<: *defaults
     steps:
       - checkout
-      - *restore_yarn_offline_mirror
-      - *restore_yarn_cache
-      - run:
-          name: Check versions and env
```

Comment:
Can we keep this step? It's great for debugging.

Reply:
I'd prefer not to with all the access tokens in there. What values are you interested in?

Reply:
People can still edit the .circleci/config.yml file and echo the env, nothing prevent them from doing it.

Reply:
Ok, maybe we can keep the dependency version then? I don't have any use case specific in mind. It's more a potential time saver than anything else. It removes the need from using the ssh mode.

Reply:
> People can still edit the .circleci/config.yml file and echo the env, nothing prevent them from doing it.

At which point we see that they done it.

> dependency version then

What version? Node, yarn are determined by the docker image anyway. The docker version is determined by CircleCi as far as I can tell.

Reply:
> At which point we see that they done it.

Sure, it's a little dissuasive.

> What version? Node, yarn are determined by the docker image anyway. The docker version is determined by CircleCi as far as I can tell.

Any version, things can go wrong. Nothing can give you more confidence than a console log.

Reply:
So we need to debug dependency without a given use case here but not anywhere else? We didn't run this in the other jobs since forever and nobody complained. Why the change of heart now?

Reply:
As you want, let's wait for a use case.